app/queries/payments_requiring_action.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class PaymentsRequiringAction
+  def initialize(user)
+    @user = user
+  end
+
+  attr_reader :user
+
+  def query
+    Spree::Payment.joins(order: [:user]).where.not(cvv_response_message: nil).
```

Comment:
It'd be great if we could make use of the scope you defined in https://github.com/openfoodfoundation/openfoodnetwork/pull/6910/files#diff-0deeea11478d8c36d2233d713f0c3e51c3f4579680a71d02d0ed32c4396fa01e. Hiding this detail it'd make things slightly more changeable.

Reply:
48ce5b8a20ff08d0d1b5f79bc8a9337748005d88 👍